Build/net45.props

```diff
@@ -72,7 +72,7 @@
     <Features>$(Features);FEATURE_ASYNC</Features>
     <Features>$(Features);FEATURE_GET_TYPE_INFO</Features>
     <Features>$(Features);FEATURE_DYNAMIC_EXPRESSION_VISITOR</Features>
-    <Features>$(Features);FEATURE_ASSEMBLYBUILDER_DEFINEDYNAMICASSEMBLY</Features>
+    <Features Condition=" '$(Mono)' != 'True' ">$(Features);FEATURE_ASSEMBLYBUILDER_DEFINEDYNAMICASSEMBLY</Features>
```

Comment:
Odd. Ah well, it's just a switch between two APIs that are equivalent anyway, we could probably completely remove it from net45.props.